configs/eslint-common-cypress.json

```diff
@@ -1,6 +1,13 @@
 {
-    "extends": ["plugin:chai-friendly/recommended", "plugin:cypress/recommended"],
-
+    "extends": [
+        "airbnb-base",
+        "./eslint-common.json",
+        "plugin:chai-friendly/recommended",
+        "plugin:cypress/recommended"
+    ],
+    "rules": {
+        "import/no-extraneous-dependencies": "off"
+    },
```

Comment:
It make sense to add it here, as we did that in local configs for Cypress in Stage / Composer.